libs/mrwebrtc/include/interop_api.h

```diff
@@ -24,6 +24,36 @@ enum class mrsOptBool : int8_t { kTrue = -1, kFalse = 0, kUnset = 0b01010101 };
 /// Report live objects to debug output, and return the number of live objects.
 MRS_API uint32_t MRS_CALL mrsReportLiveObjects() noexcept;
 
+/// Audio device modules for Windows Desktop.
+enum class mrsAudioDeviceModule : uint8_t {
+  /// Legacy module for backward compatibility. This is not recommended, unless
+  /// there is an issue with ADM2.
+  kADM1 = 1,
+
+  /// New CoreAudio module. This is the default and recommended audio module on
+  /// Windows Desktop.
+  kADM2 = 2
+};
+
+/// Enable or disable the use of the new Audio Device Module (ADM2) on Windows
+/// Desktop.
+///
+/// This mainly enables working around some unsupported devices plugged in which
+/// make the legacy module (ADM1) fail its initialization even if said devices
+/// are not later used. Therefore ADM2 is used by default, and this function is
+/// mainly used to disable it and fall back to the legacy ADM1 if needed.
+/// https://github.com/microsoft/MixedReality-WebRTC/issues/124
```

Comment:
Rewrite the comment now that we pass an enum.

> This mainly enables working around some unsupported devices plugged in which make the legacy module (ADM1) fail its initialization

This reads as if _calling this function_ enables working around unsupported devices - but it's the opposite. It should probably be something like "ADM2 is the default. ADM1 is deprecated as it fails initialization if etc.... but if you want you can force its usage by calling this function".

Reply:
Fixed, and renamed enum values to match C# too.